client/src/templates/Introduction/components/challenges.tsx

```diff
@@ -37,25 +38,44 @@ interface IsProjectBlockProps {
 const CheckMark = ({ isCompleted }: { isCompleted: boolean }) =>
   isCompleted ? <GreenPass /> : <GreenNotCompleted />;
 
-const ListChallenge = ({ challenge }: { challenge: ChallengeInfo }) => (
-  <Link to={challenge.fields.slug}>
-    <span>
-      <CheckMark isCompleted={challenge.isCompleted} />
-    </span>
-    {challenge.title}
-  </Link>
-);
-
-const CertChallenge = ({ challenge }: { challenge: ChallengeInfo }) => (
-  <Link to={challenge.fields.slug}>
-    {challenge.title}
-    <span className='map-project-checkmark'>
-      <CheckMark isCompleted={challenge.isCompleted} />
-    </span>
-  </Link>
-);
-
-export function ChallengesList({ challenges }: ChallengesProps): JSX.Element {
+const ListChallenge = ({
+  challenge,
+  onChallengeClick
+}: {
+  challenge: ChallengeInfo;
+  onChallengeClick?: () => void;
+}) => {
+  return (
+    <Link to={challenge.fields.slug} onClick={onChallengeClick}>
+      <span>
+        <CheckMark isCompleted={challenge.isCompleted} />
+      </span>
+      {challenge.title}
+    </Link>
+  );
+};
+
+const CertChallenge = ({
+  challenge,
+  onChallengeClick
+}: {
+  challenge: ChallengeInfo;
+  onChallengeClick?: () => void;
```

Comment:
This should be a required prop.

```suggestion
  onChallengeClick: () => void;
```